### PR TITLE
adds axe spray

### PR DESCRIPTION
**Pull request by MMMiracles**
## About The Pull Request

Adds a knock-off ax spray to the game that temporarily hides the stink lines from your character. Being sprayed by it applies a small amount of firestacks and spraying too much of it on yourself causes an overdose effect, where you'll be surrounded by a cloud of axe and periodically take a small amount of oxyloss unless you're wearing a gas mask or don't actually require the ability to breathe.

Credit to @Partheo / yogstation for the deodorant sprite, slightly modified.
## Why It's Good For The Game

Helps neets hide their neetness by drowning it in something arguably worse.

## Changelog
:cl: MMMiracles
add: New Nanotrasen approved Shower-In-A-Can! Hide that pesky odor from your fellow coworkers who bathe about as much as you do! Remember: Less is more!
/:cl:

### CODE REVIEW

**Comment by plapatin** [comment]
god yes

**Comment by ninjanomnom** [comment]
I'll merge this if you share the sponsorship money

**Comment by Dawson1917** [comment]
give a message to everyone around you of how pungent you smell if you spray yourself more than once in an hour

**Comment by PKPenguin321** [comment]
Is it refillable? Would be a shame to see it immediately emptied and used as a deathspray

**Comment by MMMiracles** [comment]
it has OPENCONTAINER set to 0, so you can't empty it or refill it by default.

File: code/modules/reagents/reagent_containers/spray.dm [hunk 1]
```diff
@@ -186,6 +186,19 @@
 	desc = "Gyaro brand spray tan. Do not spray near eyes or other orifices."
 	list_reagents = list(/datum/reagent/spraytan = 50)
 
+//bo spray
+/obj/item/reagent_containers/spray/spaceaxe
+	name = "nanotrasen shower-in-a-can"
```
Comment (MrPerson) [inline code/modules/reagents/reagent_containers/spray.dm]: Nanotrasen is a proper noun and should be capitalized.

Comment (PKPenguin321) [reply code/modules/reagents/reagent_containers/spray.dm]: You'll want to put a \improper at the beginning as well

**Comment by Altoids1** [comment]
https://github.com/yogstation13/Yogstation-TG/pull/4310

**Comment by Tacolizard** [comment]
god yes this is the change that will bring me back to the game finally i can roleplay as who i truly am

**Comment by 81Denton** [comment]
Tbh you could add some of this to the janitor's locker as well

It's my dream to chase down screeching ~~anime con attendants~~ Assistants and douse them with deodorant

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -903,6 +903,29 @@
...
+	taste_description = "highschool locker room"
+	glass_name = "glass of antiperspirant"
+	glass_desc = "Tastes like teen spirit."
+
+/datum/reagent/deodorant_spray/reaction_mob(mob/living/M, method=TOUCH, reac_volume)//bad for your health
+	if(method == TOUCH || method == VAPOR)
+		M.adjust_fire_stacks(reac_volume / 10)
+		..()
+
+/datum/reagent/deodorant_spray/on_mob_life(mob/living/carbon/C)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: why isn't this a part of reaction_mob? why do you need to reapply this trait every tick?

Comment (MMMiracles) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: good question

**Comment by datorangebottle** [comment]
Does it make big red text if you stand next to someone who's used an entire can on themselves?

**Comment by FloranOtten** [comment]
Does this give a separate, less powerful moodlet that indicates that you may not be smelly, but you certainly don't smell good either?

**Comment by MMMiracles** [comment]
no, it just hides the stink lines so everyone around you doesn't immediately realize how much of an fillthy person you are

**Comment by qwertyquerty** [comment]
Add ability to rubber band it then chuck it as an axe gas grenade.

**Comment by Aranclanos** [comment]
Why didn't you name the path `/obj/item/reagent_containers/spray/deodorant` ?
Also show the new sprite on the OP

**Comment by bigfatbananacyclops** [comment]
@subject smh

**Comment by subject217** [comment]
haha, jokes on you for expecting me to follow through on something mike murdock, yogstation wins once again

**Comment by MMMiracles** [comment]
harsh man, but so is life

**Pull request closed without merging**